src/modules/keycloak-administration/domain/keycloak-user.service.ts

```diff
@@ -266,12 +316,18 @@ export class KeycloakUserService {
             return { ok: false, error: new KeycloakClientError('Response is invalid') };
         }
 
+        const externalSystemIDs: ExternalSystemIDs = {};
+        if (userReprDto.attributes) {
+            externalSystemIDs.ID_ITSLEARNING = userReprDto.attributes['ID_ITSLEARNING'] as string;
+            externalSystemIDs.ID_OX = userReprDto.attributes['ID_OX'] as string;
+        }
```

Comment:
I think the attributes might actually be `string[]` (Because you can set the same attribute multiple times). Could you check if that's the case?